roles/confluent.control_center/tasks/main.yml

```diff
@@ -150,6 +150,16 @@
     owner: "{{control_center_user}}"
     mode: 0770
 
+- name: Update log4j Config for Log Cleanup
```

Comment:
So I just ran the scenario and examined c3's log4j file, it has:

```
# kafka
log4j.appender.kafka=org.apache.log4j.RollingFileAppender
log4j.appender.kafka.MaxFileSize=100MB
log4j.appender.kafka.MaxBackupIndex=10
log4j.appender.kafka.Append=true
log4j.appender.kafka.File=${confluent.controlcenter.log.dir}/control-center-kafka.log
log4j.appender.kafka.layout=org.apache.log4j.EnhancedPatternLayout
log4j.appender.kafka.layout.ConversionPattern=[%d] %p [%t] %m (%c)%n
log4j.appender.kafka.MaxFileSize=10MB
log4j.appender.kafka.MaxBackupIndex=5
log4j.appender.kafka.append=true
```

So its not quite working... and that makes sense bc this task has the wrong regex...
```
- name: Add Max Size Properties
  lineinfile:
    path: "{{log4j_file}}"
    insertafter: "log4j.appender.{{ item[0] }}=org.apache.log4j.RollingFileAppender"
    regexp: "log4j.appender.{{ item[0] }}.{{ item[1] }}"
    line: "log4j.appender.{{ item[0] }}.{{ item[1] }}"
  with_nested:
    - "{{appenders_grep.stdout_lines}}"
    - [ 'Append=true', 'MaxBackupIndex=10', 'MaxFileSize=100MB' ]
  notify: "{{handler}}"
```

The initial point was to not run that task file on c3 bc it already has rolling in there...
So maybe we can add a flag `update_ rolling_file_appender` and in this case set it to false and skip those tasks

Or we could fix that regex in that task

Reply:
To fix that task it might be
```
- name: Add Max Size Properties
  lineinfile:
    path: "{{log4j_file}}"
    insertafter: "log4j.appender.{{ item[0] }}=org.apache.log4j.RollingFileAppender"
    regexp: "log4j.appender.{{ item[0] }}.{{ item[1].split('=')[1] }}=*"
    line: "log4j.appender.{{ item[0] }}.{{ item[1] }}"
  with_nested:
    - "{{appenders_grep.stdout_lines}}"
    - [ 'Append=true', 'MaxBackupIndex=10', 'MaxFileSize=100MB' ]
  notify: "{{handler}}"
```

Reply:
adding the fix.